### DISCUSSION TITLE
Making an SVG version of the logo

### DISCUSSION THREAD

**Discussion by quazar-omega** [gogs/gogs#7744] [Proposal]
Hi! I've been trying to find the Gogs logo in SVG, but haven't seen it anywhere, just PNGs, even this issue #3385 says it's missing.  
If there isn't one, would you like me to make one?

Here's what I came up with currently in Inkscape, but it's not perfect yet:
![Vectorized Gogs logo](https://github.com/gogs/gogs/assets/56311398/43b4106a-f7a8-4f53-bd58-372d3aeab3f1)

If that's fine as a start, could you tell me what kind of gradient you're using (linear, radial, etc.) and the coordinates of the handles? I think that's mostly what's needed to refine it.
Thank you.

**Comment by GwynethLlewelyn** [comment]
Not perfect but close enough, I'd say :) I hadn't even noticed that Gogs did _not_ have a vector-based logo...

Let's hope that @unknwon takes a look at this discussion sometime and approves your design!

**Reply by quazar-omega** [reply]
> Not perfect but close enough, I'd say :)

I know, I tried what I could in 1 hour, I hope I won't be boiled alive for it haha

> I hadn't even noticed that Gogs did not have a vector-based logo...

Yeah, I didn't expect it, the various places where the logo appears are really crisp, so I believe someone may already have a vector of it, but perhaps it's made in Illustrator so it's not an SVG(?), idk.

**Comment by unknwon** [comment]
Hey @quazar-omega, thanks for asking and your effort ❤️ Gogs does have the original SVG version of the logo 😁 

![Gogs](https://github.com/gogs/gogs/assets/2946214/f9b487f0-9a80-4422-85fa-eca5fe2dbfd1)

I will take a note and make a simple brand kit repo 🤞

**Reply by quazar-omega** [reply]
Ohhh, so there *is* one! Awesome, thank you!

> I will take a note and make a simple brand kit repo 🤞

That would be cool! Especially for discoverability

**Reply by unknwon** [reply]
Here you go! https://github.com/gogs/brand-kit

**Reply by quazar-omega** [reply]
Thanks again! Will you also add the vector for the logotype?

**Reply by unknwon** [reply]
>vector for the logotype

Hmm, what kind of file extension you're referring to?

**Reply by quazar-omega** [reply]
I mean the text part of the logo, like in your [banner](https://github.com/gogs/brand-kit/blob/main/banner/gogs-banner-01.png):
![Gogs banner](https://github.com/gogs/brand-kit/blob/main/banner/gogs-banner-01.png?raw=true)

Still in SVG, it's not another type of file, sorry if it wasn't clear

**Reply by unknwon** [reply]
Gotcha! Waiting for the designer to send them to me :D

**Reply by unknwon** [reply]
Alright, uploaded! https://github.com/gogs/brand-kit/tree/main/banner

**Reply by quazar-omega** [reply]
Nice, it's all good IMO!
Thank you for being so responsive!
